### PR TITLE
docs(components): [input-number]

### PR DESCRIPTION
**Pull request by chenxch**
* Update input-number docs with new syntax.

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by holazz — APPROVED** [review]

File: docs/en-US/component/input-number.md [hunk 1]
```diff
@@ -79,38 +79,41 @@ input-number/controlled
...
+| controls                                     | whether to enable the control buttons            | ^[boolean]                                    | true      |
+| controls-position                            | position of the control buttons                  | ^[enum]`'' \| 'right'`                        | —         |
+| name                                         | same as `name` in native input                   | ^[string]                                     | —         |
+| label                                        | same as `label` in native input                  | ^[string]                                     | —         |
+| placeholder                                  | same as `placeholder` in native input            | ^[string]                                     | —         |
+| id                                           | same as `id` in native input                     | ^[string]                                     | —         |
+| value-on-clear<VersionTag version="2.2.0" /> | value should be set when input box is cleared    | ^[number] / ^[null] / ^[enum]`'min' \| 'max'` | —         |
```
Comment (jw-foss) [inline docs/en-US/component/input-number.md:103]: @chenxch maybe we should keep the old way of this since this will produce error in Chinese translations.

Comment (jw-foss) [reply docs/en-US/component/input-number.md:103]: Overall it LGTM.

Comment (tolking) [reply docs/en-US/component/input-number.md:103]: The use of `<...>` in table is easy to cause format errors after translation. I think a custom Markdown plugin can be added to deal with this problem.

Same like
 
```
^[enum]`'min' \| 'max'`
```

Comment (tolking) [reply docs/en-US/component/input-number.md:103]: @jw-foss What do you think?

Use as follows

```
^(2.2.0)

^(a11y)

^(deprecated)

//...
```

Comment (jw-foss) [reply docs/en-US/component/input-number.md:103]: @tolking agreed, maybe we should just apply this way. Do you have the plugin implementation right now?

Comment (tolking) [reply docs/en-US/component/input-number.md:103]: I can write a plugin to implement it.

**Pull request merged**